lib/checks/color/color-contrast.json

```diff
@@ -5,18 +5,14 @@
 		"ignoreUnicode": true,
 		"ignoreLength": false,
 		"boldValue": 700,
-		"boldPt": 14,
-		"nonBoldPt": 18,
+		"boldTextPt": 14,
+		"largeTextPt": 18,
 		"contrastRatio": {
 			"normal": {
-				"expected": 4.5,
-				"minThreshold": 0,
```

Comment:
My bad, I didn't realise this when I reviewed it initially, but it seems to me this could be used to do the "equal ratio" thing, where if it's 1:1, it passes. If we don't use it that way, there'd be no way to turn that behaviour off.

Reply:
I made a bad assumption in my review. Never mind this.